docs/content/metric-stores/metrics.md

```diff
@@ -55,3 +55,60 @@ It exposes the following metric-specific tags:
 See
 [here](https://github.com/alibaba/feathub/blob/master/python/feathub/metric_stores/tests/test_prometheus_metric_store.py#L122)
 for example usages.
+
+## CountMap
+
+CountMap is a group of metrics that show the number of occurrence of each
```

Comment:
How about using the following statement:

CountMap is a map that maps each unique feature value to the number of occurrences of this value.